rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -913,6 +913,26 @@ object Reduce {
         }
     }
 
+    private[this] val toUtf8Bytes: Method = new Method() {
+
+      override def apply(
+          p: Par,
+          args: Seq[Par]
+      )(implicit env: Env[Par], costAccountingAlg: CostAccountingAlg[M]): M[Par] =
+        if (args.nonEmpty) {
+          s.raiseError(ReduceError("Error: toUtf8Bytes does not take arguments"))
```

Comment:
We have some predefined constructors for method errors. For wrong number of arguments you should use `MethodArgumentNumberMismatch`

Reply:
I patterned my code after the `hexToBytes` right above `toUtf8Bytes`.  Should I fix that to also use `MethodArgumentNumberMismatch`?

Reply:
Yes please, probably the code above wasn't patched after those error classes were introduced. We need to change that.